### ISSUE TITLE
Some glyphs dropped even though they exist

### ISSUE THREAD

**Issue by EmilyBjoerk** [jaagr/polybar#1359]
```
$ polybar -l info top 2>&1 | grep "Loaded font"
polybar|info:  Loaded font "Noto Sans:pixelsize=9" (name=Noto Sans, offset=0, file=/usr/share/fonts/noto/NotoSans-Regular.ttf)
polybar|info:  Loaded font "Font Awesome 5 Free:pixelsize=10" (name=Font Awesome 5 Free, offset=1, file=/usr/share/fonts/fontawesome/Font Awesome 5 Free-Regular-400.otf)
```

```
$ grep -i font ~/.config/polybar/config
font-0 = "Noto Sans:pixelsize=9;0"
font-1 = "Font Awesome 5 Free:pixelsize=10;1"
;bar-indicator-font = 2
;bar-fill-font = 2
;bar-empty-font = 2
;bar-volume-indicator-font = 2
;bar-volume-fill-font = 2
;bar-volume-empty-font = 2	
;bar-volume-indicator-font = 2
;bar-volume-fill-font = 2
;bar-volume-empty-font = 2
```

```
$ grep -i label-volume ~/.config/polybar/config
format-volume = <label-volume> <bar-volume>
label-volume = " %percentage%%"
```
(Symbol displays correctly in urxvt and emacs, it's \uf028, volume up, gucharmap confirms symbol exists in Font Awesome 5 Free)

```
polybar top 2>&1 | grep f028
polybar|warn:  Dropping unmatched character  (U+f028)
```

```
$ emerge -pv polybar

These are the packages that would be merged, in order:

Calculating dependencies... done!
[ebuild   R    ] x11-misc/polybar-3.2.0::gentoo  USE="alsa i3wm pulseaudio -curl -ipc -mpd -network" PYTHON_TARGETS="python2_7" 0 KiB

Total: 1 package (1 reinstall), Size of downloads: 0 KiB
```

Some other glyphs render just fine, like floppy symbol.
The only thing I can see is that Font Awesome is OTF, is that an issue?

edit: Actually both fonts have glyphs for \uf082

**Comment by EmilyBjoerk** [comment]
Turns out gucharmap uses fallback fonts for glyphs unless you check "show only glyps from this font" under "View".... for some weird reason my Font Awesome is missing lots of glyps...

**Comment by EmilyBjoerk** [comment]
Fixed by #924
